src/Robo/Plugin/Traits/RoboConfigTrait.php

```diff
@@ -81,25 +82,18 @@ private function getRequiredRoboConfigValueFor(string $key)
     /**
      * Validate Robo configuration value type.
      *
-     * @param mixed $configValue
-     *   The configuration value.
-     * @param string $expectedType
-     *   The type we are expecting the value to be of.
-     * @param string $key
-     *   The key of the configuration.
-     *
-     * @return bool
-     *   TRUE if configuration value matches the expected type.
-     *
      * @throws \Robo\Exception\TaskException
      */
-    private function validateRoboConfigValueMatchesType($configValue, string $expectedType, string $key): bool
-    {
+    private function validateRoboConfigValueMatchesType(
+        mixed $configValue,
```

Comment:
TIL that `mixed` is valid.

Reply:
[It is in 8.0](https://stitcher.io/blog/new-in-php-8#new-mixed-type-rfc)! We could also maybe use a union type here like `string|array`. 🤔

Reply:
I guess it would need to be something like `string|int|float|bool|null|array`. `mixed` is probably best.
<img width="754" alt="CleanShot 2023-05-31 at 15 51 59@2x" src="https://github.com/ChromaticHQ/usher/assets/20355/50a4becf-c9d3-4d34-8a38-9868e8fa131c">